### ISSUE TITLE
Some data files hidden in `onclick` rather than `href`

### ISSUE THREAD

**Issue by skhiggins** [skhiggins/PythonTools#4]
Sometimes the file name is hidden inside of an `onclick` rather than an `href`, for example here is a piece from the source code for <http://www.inegi.org.mx/est/contenidos/proyectos/accesomicrodatos/enafin/2015/>
```html
    <div id="divam">
        <table class="EncE_Tabla" cellpadding="0" cellspacing="0" style="width:auto;">                                                        
			<tr>															
				<th class="EncE_Th" style="border-right:0px; width:75px; cursor:pointer;" onclick="descargaArchivo('/est/contenidos/Proyectos/accesomicrodatos/enafin/2015/doc/ejembd/bd_enafin_xlsx.zip')"><a href="#" id="linkDbf"  target="_top" class="menuN"><img  src="/img/botones/btn_descargaDBF.gif" align="middle" border="0" /></a></th>															
				<th class="EncE_Th" style="border-right:0px; width:75px; cursor:pointer;" onclick="descargaArchivo('/est/contenidos/Proyectos/accesomicrodatos/enafin/2015/doc/ejembd/bd_enafin_dta.zip')"><a href="#" id="linkSas"  target="_top" class="menuN"><img src="/img/botones/btn_descargaSTATA.gif" align="middle" border="0" /></a></th>
			</tr>
		</table>
    </div>
```

Also, explore the source code <view-source:http://www.inegi.org.mx/est/contenidos/proyectos/accesomicrodatos/enafin/2015/> to see if there are other ways that file types that we want to download can be hidden in fields other than `href`

**Comment by angelynaye** [comment]
fixed in the newest version

**Comment by skhiggins** [comment]
Fixed in #26
